homeassistant/components/switchbot/strings.json

```
@@ -15,9 +15,19 @@
         "data": {
           "password": "[%key:common::config_flow::data::password%]"
         }
+      },
+      "lock_key": {
+        "description": "The {name} device requires encryption key, details on how to obtain it can be found in the [documentation]({url}).",
```

```suggestion
        "description": "The {name} device requires encryption key, details on how to obtain it can be found in the documentation.",
```

Lets leave out the link since the box already has a help `?` they can click on and its one less url to maintain